app/routes.js

```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
   let assessmentMarkingRegex = new RegExp(/Marking assessments/i)
   let assessmentModeratingRegex = new RegExp(/Moderating assessments/i)
   let assessmentDesigningRegex = new RegExp(/Designing and setting assessments/i)
+  let assessmentSeniorRegex = new RegExp(/Designing and setting assessments/i)
```

Comment:
this one should be `let assessmentReviewingRegex = new RegExp(/Reviewing assessment performance/i)`